### ISSUE TITLE
A way to keep infinite precision when parsing string?

### ISSUE THREAD

**Issue by muuuub** [RubyMoney/monetize#28]
I have some varying dollar amounts:

`$10.12`
`$10.123`
`$10.1234`
`$10.12345`

 and so on.. 

Doing `Monetize.parse("10.123")` returns `10.12`. It drops anything beyond two decimal places.

Is there a way to preserve the precision of the original string?

**Comment by HaleTom** [comment]
It doesn't seem to work for me either:
```ruby
def format_code_value(money, decimal_places = 2)
  sprintf "%s %.#{decimal_places}f", money.currency, money.to_f
end

Money.infinite_precision = true

c = Money::Currency.new :usd
value = 1.234567890123456789

m1 = Monetize.parse(value, c)
m2 = Money.new(value * c.subunit_to_unit, c)

puts format_code_value(m1, 20)
puts format_code_value(m2, 20)
```
Output:
```
USD 1.22999999999999998224
USD 1.23456789012346002110
```

**Comment by HaleTom** [comment]
Given the previous comment indicates this is still an issue, can someone please re-open it?

**Comment by HaleTom** [comment]
Fixed by #68
